CI/physmon/phys_perf_mon.sh

```diff
@@ -91,8 +99,8 @@ run \
     $outdir/acts_analysis_residuals_and_pulls.root \
     $refdir/acts_analysis_residuals_and_pulls.root \
     --title "analysis_residuals_and_pulls" \
-#    -o $outdir/analysis_residuals_and_pulls.html \
-#    -p $outdir/analysis_residuals_and_pulls
+    -o $outdir/analysis_residuals_and_pulls.html \
+    -p $outdir/analysis_residuals_and_pulls
```

Comment:
This was previously producing a **lot** of plots, and we turned it off again (cf @andiwand)

Reply:
sorry, I will remove that